docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+# @format
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      - FAMLY_EMAIL=${FAMLY_EMAIL} # default to get value from host environment
+      - FAMLY_PASSWORD=${FAMLY_PASSWORD} # default to get value from host environment
+      - FAMLY_PICTURES_FOLDER=/app/pictures
+      # - FAMLY_SCHEDULE_TIME=19:00 # Uncomment if you want to run in schedule mode
+    volumes:
+      - ./pictures:/app/pictures
+    entrypoint: ["famly-fetch", "-e"]
```

Comment:
example use case, I can deploy this docker container to keep download all the files every day.